### PR TITLE
Linking ProtocolRef links to Cancer.gov

### PR DESCRIPTION
**Pull request by venglisch**

### CODE REVIEW

File: Filters/CDR0000339576.xml [hunk 1]
```diff
@@ -109,6 +109,27 @@ OCECDR-3956: Allow CitationLinks in Image Captions
  </xsl:template>
 
 
+ <!--
+ ======================================================================
+ Display a ProtocolRef
+ The element should link to ClinicalTrials.gov using the following
+ format:
+    https://clinicaltrials.gov/show/NCT00004451
```
Comment (bkline) [inline Filters/CDR0000339576.xml]: Is the comment wrong ...

File: Filters/CDR0000339576.xml [hunk 2]
```diff
@@ -109,6 +109,27 @@ OCECDR-3956: Allow CitationLinks in Image Captions
...
+ ====================================================================== -->
+ <xsl:template                   match = "ProtocolRef">
+  <a>
+   <xsl:attribute                 name = "href">
+    <!--
+    <xsl:text>https://clinicaltrials.gov/show/</xsl:text>
+    -->
+    <xsl:text>https://www.cancer.gov/clinicaltrials/</xsl:text>
+    <xsl:value-of               select = "@nct_id"/>
```
Comment (bkline) [inline Filters/CDR0000339576.xml]: ... or the code?

Comment (venglisch) [reply Filters/CDR0000339576.xml]: The comment was right for the first commit but then the requirements were modified and I forgot to update the comment.
I'll fix that.

Comment (bkline) [reply Filters/CDR0000339576.xml]: +1

You might want to remove the commented-out `<xsl:text/>` element while you're at it to eliminate any confusion about what it's doing there.

**Pull request merged**